src/main/java/no/unit/nva/model/instancetypes/journal/JournalCorrigendum.java

```diff
@@ -0,0 +1,62 @@
+package no.unit.nva.model.instancetypes.journal;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import no.unit.nva.model.pages.Range;
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
+public class JournalCorrigendum extends JournalNonPeerReviewedContent {
```

Comment:
Is this an example of a Journal Corrigendum?  (https://doi.org/10.1038/nature11848).

Is it worth putting a short explanation in the Javadoc? Something like 
"Publication correcting another publication published by the same authors. It has its own DOI"?